### DISCUSSION TITLE
Automatically arc edges

### DISCUSSION THREAD

**Discussion by filipesmedeiros** [maxGraph/maxGraph#533] [Q&A]
Hello! I was wondering if there is currently a way to use maxGraph in a way that creates automatically arced edges, like the example below?

https://github.com/user-attachments/assets/21e4397f-c8f2-41b0-964a-87157382f36f

If not, I wonder if someone can point me to the part in the code where I can contribute with this? I'm interested in helping with this! :)

Thanks

**Comment by tbouffard** [comment]
Hi, I don't have a lot of time to write a more complete answer, but here are some pointers. This is probably not what you exactly had in mind, but this seems close.

You can have a look at the former mxGraph "graph editor examples" that let you add or update "bend" points of a edge: https://jgraph.github.io/mxgraph/javascript/examples/grapheditor/www/index.html


You can have also have a look at
- ElbowConnector
- style properties of the edge: bendable, curved, rounded. You can see some in action in https://maxgraph.github.io/maxGraph/demo/?path=/story/icon-images-markers--default
- #371
- #479
